### ISSUE TITLE
Shippers Page Add calls to action

### ISSUE THREAD

**Issue by BigActual** [automatedlogistics/Theme2016#30]
Switch dead link to a call to action. add CTA in the hero image. bottom and sidebar (Banana)

**Comment by brashrebel** [comment]
I'm not following you with the dead link thing. Or the banana thing.

**Comment by BigActual** [comment]
Right, good question. Mistyped dead link and meant to say replace "No Shipper Services Found" with a call to action. In this case, "Give us a call to get started - phone number" Match font to the styling of the sidebar elements @d4mation is creating for #32 and #34.

**Comment by BigActual** [comment]
Monkeys=users, banana=cta.

**Comment by d4mation** [comment]
Is there anything unique between the sidebars for the Shippers page and the Carriers page? Wondering if this could be solved with a single Sidebar that can be managed from one location.

Referencing #32 and #34

**Comment by BigActual** [comment]
I understand what your getting at, but keep each one separate. They're serving different User Personas and will have different content. I believe this is a good place to ensure that the User can intuitively change content which makes this a good candidate for a CPT with fields to fill in. @brashrebel please jump in here.

**Comment by d4mation** [comment]
A CPT for CTAs? I guess I'm confused. 

For the Sidebars (Which were mentioned earlier in this Issue and then I tacked onto that), it seems that the only things they need are a Phone Number and a link to the "Connect" Page. Are they wanting to load CTAs within the Sidebar outside of basic links/buttons?

Are the CTAs going to be globally accessible/reusable throughout the site? That would make using a CPT make a lot of sense, but if each CTA is meant for only a certain page it may not be the best option.

Are they wanting more than one for the Shippers page? I could add a simple Repeater element to the Shippers Page and it can create CTAs automatically in a Loop and have that be stored within the Shippers Page itself. Similar to how I've structured the Home Page with the Featured Items (We can make them look different, I'm thinking implementation/interface).

I want to ensure I handle this in a way that will be expandable going toward the future.

**Comment by d4mation** [comment]
I've gone ahead and implemented basic Sidebars for #32 and #34. I separated them out so that content could be different between the two, but currently they both show just a Phone Number and a button leading to the "Connect" page.

I added the "Give us a call to get started" text in the "After Content" field that I have available on each Page so that it can be easily edited at any time.

![image](https://cloud.githubusercontent.com/assets/7770631/15295417/582d4d62-1b5e-11e6-992c-3399c8b1369c.png)

Is there anything else that should be done CTA-wise to make things easy to manage? I'm not sure how many we're wanting to have on this page and in what format.
